deeposlandia/datasets/dataset.py

```diff
@@ -0,0 +1,161 @@
+"""Encapsulates datasets in a generic Dataset class, and in some more specific
+classes that inherit from it
+
```

Comment:
useless new line

Reply:
Maybe this mother class should be in the `__init__.py` module of the `dataset` sub-package.

Reply:
New line removed, and class moved to `__init__.py`.

Reply:
Is it a good practice to put so much material in a `__init__.py` module?